app/Livewire/Admin/OrderManager.php

```
@@ -255,35 +274,187 @@ public function bulkResend()
+            $this->loadStatusCounts();
+        } catch (\Exception $e) {
+            DB::rollback();
+            $this->errorAlert('Failed to update order: '.$e->getMessage());
+        }
+    }
+
+    private function sendOrderUpdateNotification($order, $user)
+    {
+        try {
+            // Email notification
+            $subject = 'Order Status Updated';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$subject'. (Unused Code Rules)

(UnusedLocalVariable)